### PR TITLE
Added tutorial showing hot to create a simple fungible token exchange…

### PR DESCRIPTION
**Pull request by VladislavIvanov**
… contract and interact with it

### CODE REVIEW

File: sophia-token-exchange-contract-2.md [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+    
+The second approach(Scripted) will be shown in [Simple Fungible Token Exchange Smart Contract - Part 3](https://github.com/aeternity/tutorials/blob/master/sophia-token-exchange-contract-3.md) and will include:
+- building a script which will pack all of the above commands in one place;
+
+*Please start with the first approach. This will help you understand the next one.*
+
+## Manual approach
+
+### Deploying two fungible token contracts on sdk-edgenet
+Let’s refresh our memory on how to deploy contract to edgenet:
```
Comment (d-velev) [inline sophia-token-exchange-contract-2.md]: contracts*

File: sophia-token-exchange-contract-2.md [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+```
+0xa732153f7ff0134e8632086ddcfc1b87cd9cc1e7f34c7a5968b1ae5d1dac353b
+```
+The above address will be used for passing the contract address as function parameter via **aecli**.
+Interacting with the functions of the deployed second token contract will be done using the base58 format - in my case ```ct_2GdniJWzbmXqGsbodswki6eeqPHzb7evVkbPWSBWEdQWrkpapP```.
+
+#### Summary
+Our two fungible token contracts have been deployed on sdk-edgenet. 
+
+The hex addresses for the two deployed token contract in our case are:
```
Comment (d-velev) [inline sophia-token-exchange-contract-2.md]: contracts*

**Comment by d-velev** [comment]
I think the naming convention is snake case, maybe the contracts should be adjusted? @thepiwo

**Comment by thepiwo** [comment]
@VladislavIvanov @Perseverance what do you thing about snake-case naming convention?

**Comment by VladislavIvanov** [comment]
> @VladislavIvanov @Perseverance what do you thing about snake-case naming convention?

@thepiwo I agree to use the snake case as convention for variable and function names. Will update the PR.

**Pull request merged**